install_config/install/deploy_router.adoc

```diff
@@ -609,6 +609,58 @@ $ echo "    router metrics at: http://<ip>:9090/consoles/haproxy.html "
 ====
 
 
+=== Preventing Connection Failures During Restarts ===
+
+If you connect to the router while the proxy is reloading, there is a
+small chance that your connection will end up in the wrong network
+queue and be dropped.  There is kernel work going on to remove that
```

Comment:
Consider rewording this section:
This issue is being addressed. In the meantime,  it is possible to work around the problem by installing...